test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,35 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // only one object
+        assertIsNull((Object) null);
+        assertNotNull(1);
+        assertNotNull("abc");
+
+        // no objects are null
+        assertNotNull(1, "abc");
+        assertNotNull("abc", "abc", 1);
+
+        // some objects are null
+        assertIsNull("abc", null);
+        assertIsNull(1, null);
+        assertIsNull("abc", 1, null);
+
+        // all objects are null
+        assertIsNull(null, null);
+        assertIsNull(null, null, null);
```

Comment:
Very comprehensive set of tests! However, do consider the case where the object is a list, what if there are null objects within the list?